server/paymentProviders/paypal/payment.js

```diff
@@ -135,21 +135,12 @@ export async function createTransaction(order, paymentInfo) {
   return models.Transaction.createFromPayload(payload);
 }
 
-/** Add user that just donated as a backer of the collective */
-export async function addUserToCollective(order) {
-  const userId = order.createdByUser.id;
-  const donorInfo = { id: userId, CollectiveId: order.FromCollectiveId };
-  const tierInfo = { CreatedByUserId: userId, TierId: order.TierId };
-  return order.collective.findOrAddUserWithRole(donorInfo, roles.BACKER, tierInfo, { order });
-}
-
 /** Process order in paypal and create transactions in our db */
 export async function processOrder(order) {
   const paymentInfo = await executePayment(order);
   logger.info('PayPal Payment');
   logger.info(paymentInfo);
   const transaction = await createTransaction(order, paymentInfo);
-  await addUserToCollective(order);
```

Comment:
Removed this because the membership is created at the upper level, in `executeOrder` https://github.com/opencollective/opencollective-api/blob/47e075b43dd2997e944ee7f0da3f088ef4e62a0b/server/lib/payments.js#L302